recipes/jpeg/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "9b" %}
+
+package:
+    name: jpeg
+    version: {{ version }}
+
+source:
+    fn: jpegsrc.v9b.tar.gz
+    url: http://www.ijg.org/files/jpegsrc.v{{ version }}.tar.gz
+    md5: 6a9996ce116ec5c52b4870dbcd6d3ddb
+    patches:
+        - CMakeLists.txt.patch  # [win]
+
+build:
+    number: 0
+    features:
+        - vc9  # [win and py27]
+        - vc10  # [win and py34]
+        - vc14  # [win and py35]
+        
+requirements:
+    build:
+        - python  # [win]
+        - cmake  # [win]
+
+test:
```

Comment:
OK I think I've got it working under Windows. Which is the Jinja template? appveyor seems to be running all the various versions of Python.